docs/vgclog_balancedexamples.md

```diff
@@ -0,0 +1,643 @@
+
+```xml
+<gc-start id="187" type="partial gc" contextid="186" timestamp="2021-02-26T11:11:42.311">
+  <mem-info id="188" free="897581056" total="4294967296" percent="20">
+    <mem type="eden" free="0" total="2147483648" percent="0" />
+    <arraylet-primitive objects="1" leaves="4" largest="4" />
+    <remembered-set count="2749664" freebytes="160705664" totalbytes="171704320" percent="93" regionsoverflowed="1" regionsstable="12" regionsrebuilding="0"/>
+  </mem-info>
+</gc-start>
+```
+
+As expected, at the start of this increment, the eden region is full. 856 MB (897581056 B) of the total 2048 MB (2147483648 B) *eden* space is available as free memory. 856MB (897,581,056 B) of the total 4096MB (4294,967,296 B) heap is available as free memory.
+
+The status of the remembered set, a metastructure specific to openJ9 JVM generational garbage collectors, is reported by the `<remembered-set>` element. The *remembered set* metastructure keeps track of any object references that cross different regions. Each region corresponds to a single remembered set. The partial gc cycle uses and prunes the remembered sets. 
+
+At the start of the partial gc cycle, the *remembered set* is using 93% of its available memory capacity, with 153.26 MB (160705664 B) available. The `regionsoverflowed` value records the number of regions that have exceeded the non-heap memory allocation reserved for the *remembered set*. The partial GC cycle cannot reclaim memory from these overflow regions. The partial GC cycle also cannot reclaim memory from any regions whose *remembered set* is being rebuilt by an increment of a global mark cycle that is in progress.
+
+The following element, `<allocation-stats>`, records information about the state of application threads before the start of the current cycle. For this example, the thread `Group1.Backend.CompositeBackend{Tier1}.7` was the largest consumer of memory. 
+
+```xml
+<allocation-stats totalBytes="2146431360" >
+  <allocated-bytes non-tlh="96417448" tlh="2050013912" arrayletleaf="0"/>
+  <largest-consumer threadName="Group1.Backend.CompositeBackend{Tier1}.7" threadId="00000000007E9300" bytes="275750048" />
+</allocation-stats>
+```
+
+The operations of the GC increment are run and details are recorded in the `<gc-op> elements`. The logs show that this increment begins with a copy forward operation followed by a class unload. Other `balanced` partial GC cycles can also include sweep and compact operations. For details about the operations involved in `balanced` partial GC cycles, see [Balanced Policy - GC Processing](gc.md#balanced-policy). 
+
```

Comment:
it should be mentioned (or even example provided) that copy forward operation may include (although is not common) marking/tracing activity:

here is an example

```
<gc-op id="2122" type="copy forward" timems="283.814" contextid="2118" timestamp="2021-02-26T11:22:42.997">
  <memory-copied type="eden" objects="0" bytes="0" bytesdiscarded="0" />
  <memory-copied type="other" objects="6817204" bytes="194647048" bytesdiscarded="7155224" />
  <memory-cardclean objects="2203703" bytes="71708528" />
  <memory-traced type="eden" objects="1072763" bytes="24158424" />
  <memory-traced type="other" objects="0" bytes="0" />
  <regions eden="256" other="308" evacuated="308" marked="256" />
  <remembered-set-cleared processed="1079989" cleared="505972" durationms="4.722" />
  <finalization candidates="26" enqueued="2" />
  <ownableSynchronizers candidates="30316" cleared="2346" />
  <references type="soft" candidates="15469" cleared="0" enqueued="0" dynamicThreshold="11" maxThreshold="32" />
  <references type="weak" candidates="336" cleared="82" enqueued="6" />
  <stringconstants candidates="9574" cleared="0"  />
  <object-monitors candidates="175" cleared="88"  />
</gc-op>
```